app/store/tableState/table.state.ts

```
@@ -0,0 +1,119 @@
+import { create } from 'zustand';
+import type { Table } from './table.types';
+import { TableStatus, TableSize } from './table.types';
+
+interface TableStore {
+  tables: Table[] | [];
+  selectedTable: Table | null;
+  setTables: (tables: Table[]) => void;
+  setSelectedTable: (table: Table | null) => void;
+}
+
+const initialTables: Table[] = [
```

ESLint

[error] 'initialTables' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)